submissions/adeyemiJuwon/phpDatabase/orders.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+include('connect.php');
+if( isset($_GET['id']) && strlen($_GET['id']) > 0 ) {
+     $id =  $_GET['id'];
+}
+
+try{
+     $sql = "SELECT order_items.unit_price, 
+            order_items.total_amount, orders.order_date, orders.id ,customers.street_address, customers.city,customers.state, 
+            customers.country,order_items.quantity, products.name
+            FROM order_items 
+            LEFT JOIN orders ON orders.id = order_items.order_id
+            LEFT JOIN customers on orders.customer_id = customers.id
+            LEFT JOIN products ON products.id = order_items.product_id 
+            WHERE  orders.customer_id = '$id'";
+            $results = $pdo->query($sql);
+            setcookie('orders',  json_encode($results), time() + 3600);
+
+    if(isset($_COOKIE['orders'])) {
```

Comment:
What you are doing here is - If cookie is set still execute the SQL query instead of using that stored in the cookie. Kindly check what you did in your previous task

Reply:
Yes
Solved the issue 
You can help me review the best method to store the data into the cookie and retrieve it back as an array in the cookies with security in view too.
Thank you @adaugochi